src/independentTransforms.js

```diff
@@ -39,8 +39,25 @@
     },
     applyHook: {
       callback: function (values) {
+        /* TODO: set scale, rotate and translate to none once they are supported.
+                 link to bug: crbug.com/679873 */
         var transformString = internalScope.toTransform(values);
-        return {transform: transformString + ' ' + values.transform};
+        if (internalScope.webAnimationsJsTesting) {
+          return {
+            transform: transformString + ' ' + values.transform,
+            scaleForTesting: values.scale,
+            rotateForTesting: values.rotate,
+            translateForTesting: values.translate,
+            scale: '1 1 1',
+            rotate: '0deg',
+            translate: '0px'
+          };
+        }
+        return {
+          transform: transformString + ' ' + values.transform,
+          scale: '1 1 1',
+          rotate: '0deg',
+          translate: '0px'};
```

Comment:
I would rather not see so much code duplication. Instead you can store the result in a variable, append the values for testing then return it.